packages/react/src/components/tabs/Tabs.stories.tsx

```diff
@@ -112,3 +112,46 @@ export const WithCustomTheme = () => {
     </Tabs>
   );
 };
+
+export const WithCustomOnClickAction = () => {
+  const [activeTabIndex, setActiveTabIndex] = React.useState(0);
+  const [isLoading, setIsLoading] = React.useState(false);
+  const content = {
+    education: 'Daytime care for people who cannot be fully independent, such as children or elderly people.',
+    university:
+      'The objective of basic education in Finland is to support pupils&#39; growth towards humanity and ethically responsible membership of society.',
+  };
+
+  const mockLoading = () => {
+    setIsLoading(true);
+    setTimeout(() => {
+      setIsLoading(false);
+    }, 200);
+  };
+
+  const onTabClick = (activeIndex) => {
+    setActiveTabIndex(activeIndex);
```

Comment:
Hmm.. I think there's internal logic in the Tab component where it sets the active index on tab click. So Im not sure if this is needed?

Reply:
-[x] remove unnecessary state update from story onClick